### PR TITLE
Views:bloggy-app

### PR DESCRIPTION
**Pull request by Shubh-Dev**
Update!!
Changes made as suggested by the Reviewer,
Page navigation added,
Method added. 

Implemented the design from the sneak peek wireframes

For testing:
Available posts id = 1, 2, 3, 4, 5
Available users id = 5, 6, 7, 9, 10
Users who created posts = 6, 7

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+    <% @users.each do |user| %>
+        <div class="all-user-container">
+        <div class="user-img-container">
+        <img src="<%= user.photo %>" />
+        </div>
+        <div class="user-details-container">
+        <h2><%= user.name %></h2>
+        <h3>Number of Posts: <%= user.postscounter %></h3>
+        </div>
+        </div>
```
Comment (mikethreels) [inline app/views/users/index.html.erb]: - [ ] Clicking on each user should take you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,24 @@
...
+<p><%= @user.bio %></p>
+</div>
+
+<div class="user-posts">
+<%@posts.each do |post| %>
+<h4>Post #<%= post.id %></h4>
+<h4><%= post.text %></h4>
+<p>Comments: <%post.commentscounter%></p>
+<p>Likes: <%post.likescounter%></p>
+<%end%>
```
Comment (mikethreels) [inline app/views/users/show.html.erb]: - [ ] the application should only display the recent posts which you can do by using the `show_recent_posts` method you created in the user model. then you don't have to define `@posts` in the user_controller `show` action.
- [ ] there should be a button to show all the posts. which redirects to the user_post_path.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,35 +1,15 @@
...
-        <ul>
-            <li>User-1: comment - 1</li>
-            <li>User-2: comment - 2</li>
-            <li>User-3: comment - 3</li>
-            <li>User-4: comment - 4</li>
-            <li>User-5: comment - 5</li>
-        </ul>
-    
-    </div>
+    <% end %>
```
Comment (mikethreels) [inline app/views/posts/index.html.erb]: - [ ] Should use post.show_recent_comments to render 5 most recent comments in the app/views/posts/index.html.erb page

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Shubh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Shubh-Dev 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**